index.html

```diff
@@ -1,15 +1,106 @@
-        <script src=""></script>
-    </body>
+
+<head>
+    <meta charset="utf-8">
+    <title>Welcome to Sassy Addie</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/material-design-iconic-font/2.2.0/css/material-design-iconic-font.min.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS"
+        crossorigin="anonymous">
+    <link rel="stylesheet" href="assets/css/styles.css">
+</head>
+
+<body>
+    <!-- <div class="" id="fader"></div> -->
+    <div class="container text-center" id="mainContent">
+        <img id="logo" src="assets/imgs/logo.png" style="height:15%;" />
+        <div class="row set">
+            <div class="col-md-4">
+                <div class="data-box"></div>
+            </div>
+            <div class="col-md-4">
+                <div class="row justify-content-md-center timer" id="">00:00</div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">RECORD IT</button>
+                </div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">HELP ME</button>
+                </div>
+            </div>
+            <div class="col-md-4">
+                <div class="timer zmdi zmdi-play col-md-3"></div>
+                <div class="timer zmdi zmdi-pause col-md-3"></div>
+                <div class="timer zmdi zmdi-stop col-md-3"></div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">ADDIE'S INSPO</button>
+                </div>
+            </div>
+        </div>
+        <div class="row set">
+            <div class="col-md-4">
+                <div class="data-box"></div>
+            </div>
+            <div class="col-md-4">
+                <div class="row justify-content-md-center timer" id="">00:00</div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">RECORD IT</button>
+                </div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">HELP ME</button>
+                </div>
+            </div>
+            <div class="col-md-4">
+                <div class="timer zmdi zmdi-play col-md-3"></div>
+                <div class="timer zmdi zmdi-pause col-md-3"></div>
+                <div class="timer zmdi zmdi-stop col-md-3"></div>
+                <div class="row justify-content-md-center">
+                    <button type="button" class="btn btn-primary">ADDIE'S INSPO</button>
+                </div>
+            </div>
+        </div>
+        <!-- <div class="row set">
```

Comment:
In general, don't include commented code as part of PR's. Only include code that you actually want included.